src/components/CharacterOrderDialog.vue

```diff
@@ -100,8 +101,9 @@
                         selectCharacter(speakerUuid);
                         rollStyleIndex(speakerUuid, -1);
                       "
+                      aria-label="前のボイススタイル"
```

Comment:
「ボイススタイル」という言葉でいいのかはちょっと不明

Reply:
あ、「スタイル」と呼んでいます！
```suggestion
                      aria-label="前のスタイル"
```

このあたりの命名規則の一覧は作っていないのですが、基本的に[使い方](https://voicevox.hiroshiba.jp/how_to_use/)に合わせるようにしています！
どの辺りに書いていそうか調べるのご面倒だと思うので、PR頂いて違和感あったときにコメントします！